src/test/java/com/wikia/webdriver/elements/mercury/pages/ImageReview.java

```diff
@@ -0,0 +1,209 @@
+
+  @FindBy(css = ".image-review-action-buttons > button")
+  private List<WebElement> actionButtons;
+
+  @FindBy(css = ".image-review-image-container > img")
+  private List<WebElement> imagesToReview;
+
+  @FindBy(css = ".image-review-summary-dialog > .image-review-date-input > input")
+  private List<WebElement> statisticsDateInputs;
+
+
+  public ImageReview open() {
+    getUrl(String.format("%s/image-review", urlBuilder.getWikiGlobalURL()));
+    return this;
+  }
+
+  public boolean isTitleDisplayed() {
+    return this.isElementDisplayed(title);
+  }
+
+  public boolean isAlertNotificationDisplayed() {
+    return this.isElementDisplayed(alertNotification);
+  }
+
+  public boolean isTopMessageDisplayed() {
+    return this.isElementDisplayed(topMessage);
+  }
+
+  public boolean isImageInfoInModalDisplayed() {
+    return this.isElementDisplayed(imageInfoInModal);
+  }
+
+  public boolean isImageInModalDisplayed() {
+    return this.isElementDisplayed(imageInModal);
+  }
+
+  public boolean isContextLinkInModalDisplayed() {
+    return this.isElementDisplayed(contextLinkInModal);
+  }
+
+  public boolean isImageHistoryInModalDisplayed() {
+    return this.isElementDisplayed(imageHistoryInModal);
+  }
+
+  public boolean isContextLinkButtonInModalDisplayed() {
+    return this.isElementDisplayed(contextLinkButton);
+  }
+
+  public boolean isBackButtonDisplayed() {
+    return this.isElementDisplayed(backButton);
+  }
+
+  public boolean isSummaryDialogDisplayed() {
+    return this.isElementDisplayed(summaryDialog);
+  }
+
+  public boolean isShowStatisticsButtonDisplayed() {
+    return this.isElementDisplayed(showStatisticsButton);
+  }
+
+  public boolean isStatisticsDateInputDisplayed(int index) {
+    return this.isElementDisplayed(statisticsDateInputs.get(index));
+  }
+
+  public boolean isDownloadCsvDisplayed() {
+    return this.isElementDisplayed(downloadCsvButton);
+  }
+
+  public boolean isShowHistoryInputDisplayed() {
+    return this.isElementDisplayed(showHistoryInput);
+  }
+
+  public boolean isShowHistoryButtonDisplayed() {
+    return this.isElementDisplayed(showHistoryButton);
+  }
+
+  public String getAlertNotificationText() {
+    wait.forElementVisible(alertNotification);
+
+    return alertNotification.getText();
+  }
+
+  public String getCurrentStatusText() {
+    wait.forElementVisible(topMessage);
+    wait.forElementVisible(currentStatusMessage);
+
+    return currentStatusMessage.getText();
+  }
+
+  public int getNavbarButtonsNumber() {
+    wait.forElementVisible(navbar);
+
+    return navbarButtons.size();
+  }
+
+  public int getActionButtonsNumber() {
+    wait.forElementVisible(actionButtonWrapper);
+
+    return actionButtons.size();
+  }
+
+  public boolean areNavbarButtonsClickable() {
+    wait.forElementVisible(navbar);
+
+    for (WebElement navbarButton : navbarButtons) {
+      try {
+        wait.forElementClickable(navbarButton);
+      } catch(TimeoutException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)